### ISSUE TITLE
Constant HiDPI/LoDPI warning, disregarding my saved DPI settings

### ISSUE THREAD

**Issue by mikekasprzak** [pop-os/beta#181]
**How did you upgrade to 21.04? (Fresh install / Upgrade)**
Fresh install, about a week ago.

**Issue/Bug Description:**
Today I upgraded the LCD screen in my laptop by physically replacing the **12.5" 1366x768 TN** display with a **12.5" 1920x1080 IPS** display. FYI this is a common upgrade and the machine (ThinkPad X270) is very user serviceable. 

Anyway, since changing the display the UX constantly spams me with this HiDPI alert.

![Screenshot from 2021-06-17 00-22-53](https://user-images.githubusercontent.com/1570313/122331106-48036b80-cf02-11eb-8e2e-5476dc0d4b30.png)

I can click the X to dismiss the alert, but it returns immediately.

That's only the beginning.

These are the settings I'd like to use.

![Screenshot from 2021-06-16 23-55-36](https://user-images.githubusercontent.com/1570313/122331392-bba57880-cf02-11eb-9fad-865c228e58c0.png)

If I click the alert, the system seems to disregard my current settings above, setting my scale to 200%.

Finally, when I boot the system, my scaling settings are disregarded and again get bumped to 200%. I am also greeted by the previously mentioned alert.

![Screenshot from 2021-06-16 23-54-28](https://user-images.githubusercontent.com/1570313/122331860-82213d00-cf03-11eb-9829-01e6ed1561ac.png)

**Steps to reproduce (if you know):**
This only started happening once I switched to the higher resolution display (12.5" 1920x1080, or 176 PPI).

**Expected behaviour:**
To not alert me constantly and preserve my scaling choices

**Comment by leviport** [comment]
It sounds like the HiDPI daemon is misbehaving. Does everything work correctly if you turn off the HiDPI Daemon?

**Comment by mikekasprzak** [comment]
Yes confirmed, disabling the daemon fixes this. I can safely reboot now and keep my scaling settings.

It might not be important since the daemon is no longer running, but I noticed that the mode setting under HiDPI will not keep while the daemon is disabled. As an example, I've disabled it, but after a reboot it returns enabled. See below.

![Screenshot from 2021-06-17 17-35-09](https://user-images.githubusercontent.com/1570313/122475319-7c266d00-cf92-11eb-8813-ff182eaa2e5c.png)

**Comment by alexandergallen** [comment]
I'm having same issues on 20.10 with lenovo legion 5 pro 2560x1600 resolution. On restart it always puts scale back to 200% and constant LoDPI/HiDPI alerts.

Also refresh rate options, more specifically 165hz option, is not present forwhen running hybrid graphics (amd integrated) but works with nvidia discrete graphics - but I assume this is laptop related.
